packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -104,6 +152,64 @@ describe('quantic-sort', () => {
           });
       });
 
+      describe('when passing custom options as slots to the component', () => {
+        it('should display the custom options properly', () => {
+          const routeMatcherToUse =
+            param.useCase === useCaseEnum.insight
+              ? routeMatchers.insight
+              : routeMatchers.search;
+          cy.intercept('POST', routeMatcherToUse).as('searchRequest');
+          visitSort({useCase: param.useCase}, {withCustomSortOptions: true});
+
+          cy.wait('@searchRequest').then((interception) => {
+            Expect.sortCriteriaInSearchRequest(
+              interception.request.body,
+              defaultCustomOptionValue
+            );
+          });
+          Expect.displaySortDropdown(true);
+          Expect.labelContains('Sort by');
+
+          Actions.openDropdown();
+
+          Expect.sortOptionsEqual(customSortOptions);
+        });
+
+        customSortOptionsValues
+          .filter((value) => value !== defaultCustomOptionValue)
+          .forEach((value) => {
+            describe('when selecting a custom option', () => {
+              it(`should update the shown selected option to ${value}`, () => {
```

Comment:
This is a unit test too.

The part that is an e2e test is testing that the search request being sent contains the right sort criteria.

And the `Expect.logSortResults(value);`

These are e2e tests. But not that the "displayed" sort option has updated its text to the correct label, etc. This is more of something we test using unit tests.

Reply:
yeah you are right, should I leave this one also or do we want to start immediately to add jest test files in the repo? When I initially wrote these tests we had not had the discussions about unit vs E2E tests and I preferred to add a test in this existing file rather than create a whole file for 1-2 unit tests for that. If you prefer I can add those though.

Reply:
They're already written so I wouldn't remove them.

Just something to keep in mind for the future.